hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSImageFormatProtobuf.java

```diff
@@ -389,21 +496,26 @@ private void loadErasureCodingSection(InputStream in)
 
   public static final class Saver {
     public static final int CHECK_CANCEL_INTERVAL = 4096;
+    public static boolean WRITE_SUB_SECTIONS = false;
```

Comment:
I agree we should make this change.  I was originally copying the pattern set by CHECK_CANCEL_INTERVAL, but best to avoid the constants I think. I have made this change.